### PR TITLE
docs: de-dupe docs about request instructions

### LINKED ISSUE #193: [Docs]: `request` duplicates docs from the XXXHttpRequestRecipes on Starlark Types
### Where did you see an issue in our documentation?

https://docs.kurtosis.com/reference/starlark-instructions#request

### Description of the issue

Before we had the "Starlark Types" dedicated page, we were inlining the Get/PostHttpRequestRecipe params under `request` SL instruction. This legacy is left over in what's there now:
```
get_request_recipe = GetHttpRequestRecipe(
    # The port ID that is the server port for the request
    # MANDATORY
    port_id = "my_port",

    # The endpoint for the request
    # MANDATORY
    endpoint = "/endpoint?input=data",

    # The extract dictionary takes in key-value pairs where:
    # Key is a way you refer to the extraction later on
    # Value is a 'jq' string that contains logic to extract from response body
    # OPTIONAL
    extract = {
        "extracted-field": ".name.id",
    },
)
```
For all places in the `request` docs, we should direct the users to the official docs of the Get/PostHttpRequestRecipe on the Starlark Types page for full details, and remove the explanatory comments (e.g. `MANDATORY`) on the properties of the recipes under the `request` section.

### PR DESCRIPTION
**Pull request by leeederek**
## Description:
Fixes #193 

## Is this change user facing?
YES

## References (if applicable):
https://github.com/kurtosis-tech/kurtosis/issues/193

### CODE REVIEW

File: docs/docs/reference/starlark-instructions.md [hunk 1]
```diff
@@ -333,39 +333,22 @@ The previous `GetHttpRequestRecipe` type `GetHttpRequestRecipe(service_name = "m
...
     # OPTIONAL (Defaults to [200, 201, ...])
     acceptable_codes = [200, 500], # Here both 200 and 500 are valid codes that we want to accept and not fail the instruction
     
-    # If False, instruction will never fail based on code (acceptable_codes will be ignored).
+    # If false, instruction will never fail based on code (acceptable_codes will be ignored).
     # You can chain this call with assert to check codes after request is done.
-    # OPTIONAL (Defaults to False)
-    skip_code_check = False,
+    # OPTIONAL (defaults to false)
+    skip_code_check = false,
```
Comment (mieubrisse) [inline docs/docs/reference/starlark-instructions.md:349]: This shouldn't change; the false value in Starlark and Python are indeed  `False` with uppercase

**Review by mieubrisse — APPROVED** [review]

File: docs/docs/reference/starlark-instructions.md [hunk 2]
```diff
@@ -333,39 +333,22 @@ The previous `GetHttpRequestRecipe` type `GetHttpRequestRecipe(service_name = "m
...
     # MANDATORY
-    recipe = get_request_recipe,
+    recipe = request_recipe,
     
     # If the recipe returns a code that does not belong on this list, this instruction will fail.
     # OPTIONAL (Defaults to [200, 201, ...])
     acceptable_codes = [200, 500], # Here both 200 and 500 are valid codes that we want to accept and not fail the instruction
     
-    # If False, instruction will never fail based on code (acceptable_codes will be ignored).
+    # If false, instruction will never fail based on code (acceptable_codes will be ignored).
```
Comment (leeederek) [inline docs/docs/reference/starlark-instructions.md]:
```suggestion
    # If False, instruction will never fail based on code (acceptable_codes will be ignored).
```

File: docs/docs/reference/starlark-instructions.md [hunk 3]
```diff
@@ -333,39 +333,22 @@ The previous `GetHttpRequestRecipe` type `GetHttpRequestRecipe(service_name = "m
...
     # You can chain this call with assert to check codes after request is done.
-    # OPTIONAL (Defaults to False)
-    skip_code_check = False,
+    # OPTIONAL (defaults to false)
```
Comment (leeederek) [inline docs/docs/reference/starlark-instructions.md]:
```suggestion
    # OPTIONAL (defaults to False)
```

File: docs/docs/reference/starlark-instructions.md (hunk 1 above)
Comment (leeederek) [reply docs/docs/reference/starlark-instructions.md:349]:
```suggestion
    skip_code_check = False,
```

**Pull request merged**